packages/plugins/other/jsdoc-operations/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@graphql-codegen/jsdoc",
+  "version": "1.8.3",
```

Comment:
Can you please bump it to the latest? `1.10.0`

Reply:
I'll leave this comment unresolved so I can bump it once if this PR get's closer to a merge 🙂